lib/konbata/models/upload_course.rb

```diff
@@ -85,6 +103,12 @@ def upload_content(filename)
       puts "Uploading: #{name}"
       upload_to_s3(migration, filename)
       puts "Done uploading: #{name}"
+
+      if File.exist?(source_for_imscc)
```

Comment:
This probably isn't optimal since we could decide to handle other zip files but for now it works. If we want to make something more suitable I would suggest looking into the manifest and checking for whether it's a scorm package or not.